src/keyboardNavigation.ts

```diff
@@ -0,0 +1,82 @@
+import { isDeepestVisible, isAncestor, isEmptyObject } from './utils';
+
+function findDeepestVisibleChild(node: HTMLElement): HTMLElement {
```

Comment:
`findDeepestVisibleChild` sounds like a full graph traversal. i think what you're doing here is more like `findLastVisibleDescendant` (child is fine too i just tend towards really precise terminology for vanilla js) since it may not be the deepest visible, it's just visually last in the list

Reply:
Ahh I see what you mean. No qualms with swapping to `findLastVisibleDescendant`

Changes applied